lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php

```diff
@@ -128,4 +129,29 @@ private function removeUnauthorizedStatuses($statuses) {
     return $statuses;
   }
 
+  /**
+   * Remove Reportback Items with unauthorized statuses based on user permission.
+   *
+   * @param  array $data
+   * @return mixed
+   * @throws Exception
+   */
+  private function removeUnauthorizedResults($data) {
```

Comment:
this looks very similar to the pr you just merged... can it not be the same func?